src/components/Timeline/Playbook/Play.js

```diff
@@ -2,9 +2,16 @@ import React from 'react'
 import { connect } from 'react-redux'
 import FlatButtonStyled from 'components/elements/FlatButtonStyled'
 import { fillTemplate, publishEvent } from 'services/playbookService'
+import config from 'config'
+import appInsights from 'src/appInsights'
+
+var trackLinkClick = (name) => {
+  // eslint-disable-next-line no-undef
```

Comment:
Shouldn't need the eslint directive now that appInsights is imported rather than assumed to be in the global scope